Makefile.toml

```diff
@@ -115,10 +115,7 @@ description = "Runs prettier to check appropriate JS code format."
 workspace = false
 script = [
 '''
-jsfiles=$(git diff --no-ext-diff --name-only --diff-filter=ACM "*.js" "*.jsx" | tr '\n' ' ')
-[ -z "$jsfiles" ] && exit 0
-
-echo "$jsfiles" | xargs ./api_tests/node_modules/.bin/prettier --write
+find . -not -path '*/node_modules/*' -name '*.js' | xargs ./api_tests/node_modules/.bin/prettier --write
```

Comment:
❓Should we run this only on the JS files in `api_tests`?

Reply:
I think it should be done for all JS files, even if we decide to have more JavaScript in the codebase.

I also wanted to do it for the Json files we have but @LLFourn was against it.